### PR TITLE
Added Gitter.IM to the options to communicate

### PR DESCRIPTION
**Pull request by Wajeed-msft**

### CODE REVIEW

File: msteams-platform/feedback.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ The Microsoft Teams product team responds to the developer community across seve
...
 |        **Documentation issues**        |                                                                                                                                                                      Choose `This page` option in the **Submit and view feedback for** section, available at bottom of respective documentation to open an [issue](https://github.com/MicrosoftDocs/msteams-docs/issues) on the Microsoft Teams documentation GitHub repo.                                                                                                                                                                      |
 |       **Documentation updates**        | Select the Edit link on an article and submit a pull request to the [Microsoft Teams documentation GitHub](https://github.com/MicrosoftDocs/msteams-docs) repo.                                                                                                                                                                      |
+|       **Community chat group**        | Informal community chat room for [Microsoft Teams app developers](https://gitter.im/OfficeDev/MicrosoftTeamsAppDev)|
```
Comment (surbhigupta12) [inline msteams-platform/feedback.md:21]: To maintain parallelism, suggest rephrasing to the following:

```suggestion
|       **Community chat group**        | Select **SIGN IN TO START TALKING** to join the informal community chat room for [Microsoft Teams app developers](https://gitter.im/OfficeDev/MicrosoftTeamsAppDev).|
```

**Review by surbhigupta12 — CHANGES_REQUESTED** [review]
Minor comment. Please verify and update.

**Review by surbhigupta12 — APPROVED** [review]
The changes seem fine. Approving it.

File: msteams-platform/feedback.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ The Microsoft Teams product team responds to the developer community across seve
...
 |       **Documentation updates**        | Select the Edit link on an article and submit a pull request to the [Microsoft Teams documentation GitHub](https://github.com/MicrosoftDocs/msteams-docs) repo.                                                                                                                                                                      |
+|       **Community chat group**        | Select **SIGN IN TO START TALKING** to join the informal community chat room for [Microsoft Teams app developers](https://gitter.im/OfficeDev/MicrosoftTeamsAppDev) if you are logging in for the first time. If you are already logged in, select **JOIN ROOM**.|
```
Comment (surbhigupta12) [inline msteams-platform/feedback.md:21]: Please change to:

```suggestion
|       **Community chat group**        | Select **SIGN IN TO START TALKING** to join the informal community chat room for [Microsoft Teams app developers](https://gitter.im/OfficeDev/MicrosoftTeamsAppDev), if you are logging in for the first time. Select **JOIN ROOM**, if you are already logged in. |
```

**Review by surbhigupta12 — CHANGES_REQUESTED** [review]
Minor change, please verify.

**Review by KirtiPereira — DISMISSED** [review]
@surbhigupta12 - approved

**Review by surbhigupta12 — APPROVED** [review]
Thank you for incorporating the comments.

**Comment by surbhigupta12** [comment]
Merging this file.

**Pull request merged**